client/blocks/login/social.jsx

```diff
@@ -213,7 +213,7 @@ class SocialLoginForm extends Component {
 
 					<p className="login__social-tos">
 						{ this.props.translate(
-							"If you continue with Google or Apple and don't already have a WordPress.com account, you" +
+							"xxxIf you continue with Google or Apple and don't already have a WordPress.com account, you" +
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **23** times:
`translate( 'If you continue with Google or Apple and don't already have a WordPress.com account, you are creating an account and you agree to our {{a}}Terms of Service{{/a}}.' )` *ES Score: 7*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/teamcity-translations-job)